src/v0/destinations/tune/transform.test.js

```diff
@@ -0,0 +1,177 @@
+const { InstrumentationError } = require('@rudderstack/integrations-lib');
+const { responseBuilder, processEvent } = require('./transform');
+
+describe('responseBuilder', () => {
```

Comment:
We  generally write component tests for the integration `unit` tests like these we generally do for util functions